### ISSUE TITLE
[Login/Logout] I CAN NOT LOG IN / NO PUEDO INICIAR SESION

### ISSUE THREAD

**Issue by F9RogerF** [JonasNilson/idle_master_extended#184]
Tengo 2 cuentas de Steam y quiero cambiar la cuenta de Steam  y cuando hago clic en cerrar sesión e iniciar sesión de nuevo, automáticamente se registra en mi cuenta predeterminada, así que no puedo cambiar la información de inicio de sesión. Alguien me podria dar una solucion por favor???

**Comment by alejoxd12313** [comment]
tengo el mismo error no lo puedo solucionar

**Comment by JonasNilson** [comment]
> I have 2 Steam accounts and I want to change the Steam account and when I click sign out and sign in again it automatically logs into my default account so I can't change the login information. Could someone give me a solution please ??? (Google Translate)

It looks like the "sign-out" functionality have changed. I am experiencing the same issue when pressing "Sign Out" in Idle Master Extended.

One solution is to delete the application data saved under `%USERPROFILE%\AppData\Local\IdleMasterExtended\` and restarting Idle Master Extended:
* ![image](https://user-images.githubusercontent.com/5990550/114883418-31a73980-9e05-11eb-933b-123cea5883ae.png)

That reset the stored cookie data and had me sign in again:
* ![image](https://user-images.githubusercontent.com/5990550/114883499-44217300-9e05-11eb-995b-e6ff9d68f0a1.png)

**Comment by zRenzo** [comment]
Doesnt work for me :( cant change to my second account.

**Comment by enzosanchezc** [comment]
Hey @JonasNilson what's weird about this bug is that it only happens on some _accounts_, it doesn't seem to depend of the system. Also, it started happening after the last update, at least in my case.

**Comment by Realal** [comment]
Thanks for sending me to this thread. 

Hi all, yes, same problem here. To add to this thread and maybie give some hints for a solution, I will explain my situation. 

So like the posters here, I have 2 accounts, and I cannot change for the account I want. I want to idle my first account, but the program keeps using my 2nd account, and I do'nt get the option to change this.

I have deleted every single cookie manually (so even these cookies one cannot delete with the historic erasing option), every other history elements stored in my browser, I deleted this as well %USERPROFILE%\AppData\Local\IdleMasterExtended\ , and the program of course.

But there's some history left, because the program, when I install it again, keeps using my other account, and I cannot change that. The program stored what I typed for my other account.

What I EXACTLY get? A BLANK window pops up when I sign in (actually in the upper top I can almost read the name of my second account, barely), and it automatically signs to my other account. I cannot change this because the window is BLANK, and it signs pretty fast.

Just like that.

+  I also tried to type my steamid cookie (which section?), + steamSecure (where?) and parental (how?), where I was asked to do so (the cross, in the lower left), to no avail.

+ I also don't have the option to type my username and steam password, well to do this manually. This is odd because I read that there's normally this option left in the program options.

Cheers.

**Comment by enzosanchezc** [comment]
Sorry, i don't have any other solution, we'll have to wait till Jonas answers to see if he's got any hint.

I'll also add this image becouse i forgot to mention what items to delete

![image](https://user-images.githubusercontent.com/29806127/117484579-a4f02700-af3d-11eb-97cd-00aff95167a7.png)

**Comment by Realal** [comment]
Thank you for your suggestion, I dit it before posting, that's the first thing I did. 

And I deleted all of this + the cookies you cannot delete by this mean. 

In my google browser, this is in "Paramètres" then in "Confidentialité et sécurité" then in "Cookies et autres données de site" then in "Ensemble des cookies et données de site" and finally "Tout supprimer". I am french. Google translate may help.
